src/main/java/dev/dbos/transact/execution/DBOSExecutor.java

```diff
@@ -489,7 +488,7 @@ public <T> T syncWorkflow(
     return runWorkflowAndSaveResult(target, args, function, workflowId);
   }
 
-  public <T> WorkflowHandle<T> submitWorkflow(
+  public <T> WorkflowHandle<T, ?> submitWorkflow(
```

Comment:
this really feels like we should specify both return and exception types or neither